src/main/xerus/monstercat/api/Cache.kt

```diff
@@ -11,14 +11,14 @@ import xerus.monstercat.Settings
 import xerus.monstercat.Sheets
 import xerus.monstercat.api.response.Release
 import xerus.monstercat.api.response.ReleaseList
-import xerus.monstercat.api.response.ReleaseResponse
+import xerus.monstercat.api.response.ReleaseListResponse
 import xerus.monstercat.api.response.Track
 import xerus.monstercat.cacheDir
 import xerus.monstercat.downloader.CONNECTSID
 import xerus.monstercat.globalDispatcher
 import java.io.File
 
-private const val cacheVersion = 5
+private const val cacheVersion = 10
```

Comment:
wut? I mean it doesn't really matter, but why bump it up so much?

Reply:
I figured we'd start fresh with something round with v2. As you prefer.

Reply:
oh well, if you see it like that, why not ^^